docs/source/get-started/packaging-for-deployment.rst

```diff
@@ -4,44 +4,46 @@ Packaging for deployment
 
 BentoML provides a standardized format called Bentos for packaging AI/ML services. A Bento includes all the components required to run AI services, such as source code, Python dependencies, model artifacts, and configurations. This ensures your AI services are consistent and reproducible across different environments.
 
-Common build options
---------------------
+Define the runtime environment
+------------------------------
 
-Build options refer to a set of configurations for building a BentoML project into a Bento. These options can be defined in a ``pyproject.toml`` file under the ``[tool.bentoml.build]`` section or a YAML file (typically named ``bentofile.yaml``).
+Before building a Bento, you need to define the runtime environment of it. Here's an example:
 
-Here's an example ``bentofile.yaml`` file for :doc:`/get-started/hello-world`.
+.. code-block:: python
+    :caption: `service.py`
 
-.. code-block:: yaml
+    import bentoml
 
-    service: "service.py:Summarization"
-    include:
-      - "*.py"
-    python:
-      packages:
-        - torch
-        - transformers
+    my_image = bentoml.images.PythonImage(python_version="3.11") \
+        .python_packages("torch", "transformers")
 
-Key fields:
+    @bentoml.service(
+        image=my_image,
+        envs=[
+            {"name": "HF_TOKEN", "value": "your_hf_token"},
```

Comment:
You can omit the `value` here and the values can be provided at deployment time.